scripts/customize_build_env.sh

```diff
@@ -350,52 +343,52 @@ EOF
                 ;;
             "zen3")
 cat <<EOF  >> ${yaml}
-  compilers:
-  - compiler:
-      spec: rocmcc@5.4.3
-      paths:
-        cc: cc
-        cxx: CC
-        f77: ftn
-        fc: ftn
-      flags: {}
-      operating_system: rhel8
-      target: any
-      modules:
-      - PrgEnv-amd
-      - amd/5.4.3
-      environment: {}
-      extra_rpaths: []
+  # compilers:
+  # - compiler:
+  #     spec: rocmcc@5.4.3
+  #     paths:
+  #       cc: cc
+  #       cxx: CC
+  #       f77: ftn
+  #       fc: ftn
+  #     flags: {}
+  #     operating_system: rhel8
+  #     target: any
+  #     modules:
+  #     - PrgEnv-amd
+  #     - amd/5.4.3
+  #     environment: {}
+  #     extra_rpaths: []
```

Comment:
what will happen in this case? could it choose the wrong compiler?

Reply:
The compiler will be found outside of the environment.